pytest_docker_postgres/__init__.py

```diff
@@ -65,3 +78,15 @@ def db_engine(in_docker_compose, docker_services):
     create_database(url)
     yield create_engine(url)
     drop_database(url)
+
+
+@pytest.fixture(scope="function")
+def schema_db_engine(db_engine, load_database):
+    connection = db_engine.connect()
+    sql_files = sorted(glob.glob(os.path.join(load_database, "**/*.sql"), recursive=True))
+    for file_path in sql_files:
+        file = open(file_path)
+        trans = connection.begin()
```

Comment:
i'm not sure why this is necessary though, in theory sqlalchemy is always in autocommit mode
also it should be `with` as well